Doc/doxyfile

```diff
@@ -812,11 +812,8 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = Scan/utkscan/core/include/pugiconfig.hpp  \
-                       Scan/utkscan/core/include/pugixml.hpp \
-                       Scan/utkscan/core/source/pugixml.cpp \
-                       Scan/utkscan/include/MersenneTwister.hpp \
-                       README.md \
+EXCLUDE                = Analysis/Utkscan/core/include/MersenneTwister.hpp \
```

Comment:
This is the header that contains an implementation of the MersenneTwister algorithm. It was implemented to provide the random numbers that we add to the energies from pixie. Its original purpose was because C++ didn't have any useful built in random number generators. 

Since we've adopted C++ Std 11 we could remove this file, or at least move it to the base Resources folder. However, I've just not gotten around to tackling that problem.